version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -156,6 +164,10 @@ public String getConsoleSlot() {
         return consoleSlot;
     }
 
+    public String getCoolName() {
```

Comment:
could you rename the method to `getProductBanner()` and add javadoc that this property can be `null`?

Reply:
removing the null possibility